BLL/BLL.csproj

```diff
@@ -5,6 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.7" />
+    <PackageReference Include="MockQueryable.Core" Version="3.1.3" />
```

Comment:
Those packages are not needed for BLL project

Reply:
Removed them.